src/objects/Mesh.js

```diff
@@ -178,13 +178,13 @@ Mesh.prototype = Object.assign( Object.create( Object3D.prototype ), {
 
 		}
 
-		function checkBufferGeometryIntersection( object, raycaster, ray, position, uv, a, b, c ) {
+		function checkBufferGeometryIntersection( object, material, raycaster, ray, position, uv, a, b, c ) {
```

Comment:
> Maybe you can refactor at some point... if you want...

Not sure i wanna do this 😅. 

For this PR i'd like a solution with as few changes as possible.